### ISSUE TITLE
Not Able to Login to the CVAT UI after adding serverless and analytics feature in my existing cvat 

### ISSUE THREAD

**Issue by shreya-17** [opencv/cvat#4585]
### My actions before raising this issue
I have cloned this repo and by following the instructions I have installed cvat and able to succesfully login to the cvat console in my macbook.

#### Problem
I want to add analytics and serverless feature in my existing cvat console I followed the documentation for anaytics and serverless but now I am not able to login to the console getting this error:

**"Error: Request failed with status code 502. "Bad Gateway".cvat ui login"**

### Current Behaviour
**"Error: Request failed with status code 502. "Bad Gateway".cvat ui login"**

What is wrong with my approach I am following this document.
[https://openvinotoolkit.github.io/cvat/docs/administration/advanced/]

**Comment by shreya-17** [comment]
**MacBook-Pro cvat % docker logs cvat**
2022-04-27 18:53:19,949 INFO RPC interface 'supervisor' initialized
2022-04-27 18:53:19,955 CRIT Server 'unix_http_server' running without any HTTP authentication checking
2022-04-27 18:53:19,956 INFO supervisord started with pid 1
2022-04-27 18:53:20,972 INFO spawned: 'ssh-agent' with pid 11
2022-04-27 18:53:20,994 INFO spawned: 'clamav_update' with pid 13
2022-04-27 18:53:21,030 INFO spawned: 'git_status_updater' with pid 17
2022-04-27 18:53:21,077 INFO spawned: 'rqscheduler' with pid 19
2022-04-27 18:53:21,082 INFO spawned: 'rqworker_default_0' with pid 20
2022-04-27 18:53:21,085 INFO spawned: 'rqworker_default_1' with pid 22
2022-04-27 18:53:21,092 INFO spawned: 'rqworker_low' with pid 23
2022-04-27 18:53:21,108 INFO spawned: 'runserver' with pid 26
2022-04-27 18:53:21,504 DEBG fd 14 closed, stopped monitoring <POutputDispatcher at 274914896480 for <Subprocess at 274914622432 with name clamav_update in state STARTING> (stderr)>
2022-04-27 18:53:21,524 DEBG fd 10 closed, stopped monitoring <POutputDispatcher at 274914623104 for <Subprocess at 274914622432 with name clamav_update in state STARTING> (stdout)>
2022-04-27 18:53:21,740 DEBG 'ssh-agent' stdout output:
SSH_AUTH_SOCK=/tmp/ssh-agent.sock; export SSH_AUTH_SOCK;
echo Agent pid 11;

2022-04-27 18:53:21,741 INFO exited: clamav_update (exit status 0; not expected)
2022-04-27 18:53:21,743 DEBG received SIGCHLD indicating a child quit
2022-04-27 18:53:21,748 DEBG 'ssh-agent' stderr output:
debug2: fd 3 setting O_NONBLOCK

2022-04-27 18:53:22,001 DEBG 'rqworker_default_1' stderr output:
wait-for-it.sh: waiting for cvat_redis:6379 without a timeout

2022-04-27 18:53:22,002 INFO success: ssh-agent entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-04-27 18:53:22,002 INFO success: git_status_updater entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-04-27 18:53:22,015 DEBG 'rqworker_default_0' stderr output:
wait-for-it.sh: waiting for cvat_redis:6379 without a timeout

2022-04-27 18:53:22,029 DEBG 'rqscheduler' stderr output:
wait-for-it.sh: waiting for cvat_redis:6379 without a timeout

2022-04-27 18:53:22,033 DEBG 'git_status_updater' stderr output:
wait-for-it.sh: waiting for cvat_redis:6379 without a timeout

2022-04-27 18:53:22,071 DEBG 'runserver' stderr output:
wait-for-it.sh: waiting for cvat_db:5432 without a timeout

2022-04-27 18:53:22,071 INFO success: rqscheduler entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-04-27 18:53:22,154 DEBG 'rqworker_low' stderr output:
wait-for-it.sh: waiting for cvat_redis:6379 without a timeout

2022-04-27 18:53:22,161 INFO success: rqworker_default_0 entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-04-27 18:53:22,169 INFO success: rqworker_default_1 entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-04-27 18:53:22,170 INFO success: rqworker_low entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-04-27 18:53:22,170 INFO success: runserver entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-04-27 18:53:22,195 DEBG 'rqworker_default_1' stderr output:
wait-for-it.sh: cvat_redis:6379 is available after 0 seconds

2022-04-27 18:53:22,206 DEBG 'rqworker_default_0' stderr output:
wait-for-it.sh: cvat_redis:6379 is available after 0 seconds

2022-04-27 18:53:22,263 DEBG 'rqscheduler' stderr output:
wait-for-it.sh: cvat_redis:6379 is available after 0 seconds

2022-04-27 18:53:22,271 DEBG 'rqworker_default_1' stderr output:
bash: cannot set terminal process group (-1): Inappropriate ioctl for device

2022-04-27 18:53:22,272 DEBG 'rqworker_default_1' stderr output:
bash: no job control in this shell

2022-04-27 18:53:22,290 DEBG 'runserver' stderr output:
wait-for-it.sh: cvat_db:5432 is available after 0 seconds

2022-04-27 18:53:22,308 DEBG 'rqworker_default_0' stderr output:
bash: cannot set terminal process group (-1): Inappropriate ioctl for device

2022-04-27 18:53:22,320 DEBG 'rqworker_default_0' stderr output:
bash: no job control in this shell

2022-04-27 18:53:22,366 DEBG 'git_status_updater' stderr output:
wait-for-it.sh: cvat_redis:6379 is available after 0 seconds

2022-04-27 18:53:22,415 DEBG 'runserver' stderr output:
bash: cannot set terminal process group (-1): Inappropriate ioctl for device

2022-04-27 18:53:22,416 DEBG 'runserver' stderr output:
bash: no job control in this shell

2022-04-27 18:53:22,421 DEBG 'rqscheduler' stderr output:
bash: cannot set terminal process group (-1): Inappropriate ioctl for device
bash: no job control in this shell

2022-04-27 18:53:22,459 DEBG 'rqworker_low' stderr output:
wait-for-it.sh: cvat_redis:6379 is available after 0 seconds

2022-04-27 18:53:22,487 DEBG 'git_status_updater' stderr output:
bash: cannot set terminal process group (-1): Inappropriate ioctl for device

2022-04-27 18:53:22,488 DEBG 'git_status_updater' stderr output:
bash: no job control in this shell

2022-04-27 18:53:22,561 DEBG 'rqworker_low' stderr output:
bash: cannot set terminal process group (-1): Inappropriate ioctl for device

2022-04-27 18:53:22,562 DEBG 'rqworker_low' stderr output:
bash: no job control in this shell

2022-04-27 18:53:23,576 INFO spawned: 'clamav_update' with pid 200
2022-04-27 18:53:24,186 DEBG fd 10 closed, stopped monitoring <POutputDispatcher at 274914896624 for <Subprocess at 274914622432 with name clamav_update in state STARTING> (stdout)>
2022-04-27 18:53:24,187 DEBG fd 16 closed, stopped monitoring <POutputDispatcher at 274914896432 for <Subprocess at 274914622432 with name clamav_update in state STARTING> (stderr)>
2022-04-27 18:53:24,188 INFO exited: clamav_update (exit status 0; not expected)
2022-04-27 18:53:24,188 DEBG received SIGCHLD indicating a child quit
2022-04-27 18:53:26,218 INFO spawned: 'clamav_update' with pid 206
2022-04-27 18:53:26,927 DEBG fd 10 closed, stopped monitoring <POutputDispatcher at 274914623104 for <Subprocess at 274914622432 with name clamav_update in state STARTING> (stdout)>
2022-04-27 18:53:26,971 DEBG fd 16 closed, stopped monitoring <POutputDispatcher at 274914896576 for <Subprocess at 274914622432 with name clamav_update in state STARTING> (stderr)>
2022-04-27 18:53:26,973 INFO exited: clamav_update (exit status 0; not expected)
2022-04-27 18:53:26,973 DEBG received SIGCHLD indicating a child quit
2022-04-27 18:53:30,119 INFO spawned: 'clamav_update' with pid 209
2022-04-27 18:53:30,423 DEBG fd 10 closed, stopped monitoring <POutputDispatcher at 274914736112 for <Subprocess at 274914622432 with name clamav_update in state STARTING> (stdout)>
2022-04-27 18:53:30,424 DEBG fd 16 closed, stopped monitoring <POutputDispatcher at 274914896672 for <Subprocess at 274914622432 with name clamav_update in state STARTING> (stderr)>
2022-04-27 18:53:30,425 INFO exited: clamav_update (exit status 0; not expected)
2022-04-27 18:53:30,427 DEBG received SIGCHLD indicating a child quit
2022-04-27 18:53:31,440 INFO gave up: clamav_update entered FATAL state, too many start retries too quickly
2022-04-27 18:53:32,359 DEBG 'ssh-agent' stderr output:
debug2: fd 4 setting O_NONBLOCK

2022-04-27 18:53:32,594 DEBG 'ssh-agent' stderr output:
debug2: fd 4 setting O_NONBLOCK

2022-04-27 18:53:32,596 DEBG 'ssh-agent' stderr output:
debug1: process_message: socket 1 (fd=4) type 11

2022-04-27 18:53:32,663 DEBG 'ssh-agent' stderr output:
debug2: fd 4 setting O_NONBLOCK

2022-04-27 18:53:32,697 DEBG 'ssh-agent' stderr output:
debug1: process_message: socket 1 (fd=4) type 17

2022-04-27 18:53:32,736 DEBG 'ssh-agent' stderr output:
debug1: process_add_identity: add ssh-rsa SHA256:g1TYNpJh/J5lofS6qRaapsSclKpUyl7TaT06k8BrY10 "django@354771a174ee" (life: 0) (confirm: 0) (provider: none)

2022-04-27 18:53:32,739 DEBG 'rqworker_default_1' stderr output:
Identity added: /home/django/.ssh/id_rsa (django@354771a174ee)

2022-04-27 18:53:32,902 DEBG 'ssh-agent' stderr output:
debug2: fd 4 setting O_NONBLOCK

2022-04-27 18:53:32,934 DEBG 'ssh-agent' stderr output:
debug1: process_message: socket 1 (fd=4) type 17

2022-04-27 18:53:32,959 DEBG 'ssh-agent' stderr output:
debug1: process_add_identity: add ssh-rsa SHA256:g1TYNpJh/J5lofS6qRaapsSclKpUyl7TaT06k8BrY10 "django@354771a174ee" (life: 0) (confirm: 0) (provider: none)

2022-04-27 18:53:32,982 DEBG 'rqscheduler' stderr output:
18:53:32 Registering birth

2022-04-27 18:53:33,025 DEBG 'ssh-agent' stderr output:
debug2: fd 4 setting O_NONBLOCK

2022-04-27 18:53:33,026 DEBG 'ssh-agent' stderr output:
debug1: process_message: socket 1 (fd=4) type 11

2022-04-27 18:53:33,141 DEBG 'ssh-agent' stderr output:
debug2: fd 4 setting O_NONBLOCK

2022-04-27 18:53:33,188 DEBG 'ssh-agent' stderr output:
debug1: process_message: socket 1 (fd=4) type 17

2022-04-27 18:53:33,197 DEBG 'ssh-agent' stderr output:
debug1: process_add_identity: add ssh-rsa SHA256:g1TYNpJh/J5lofS6qRaapsSclKpUyl7TaT06k8BrY10 "django@354771a174ee" (life: 0) (confirm: 0) (provider: none)

2022-04-27 18:53:33,283 DEBG 'ssh-agent' stderr output:
debug2: fd 4 setting O_NONBLOCK

2022-04-27 18:53:33,286 DEBG 'ssh-agent' stderr output:
debug1: process_message: socket 1 (fd=4) type 11

2022-04-27 18:53:33,448 DEBG 'ssh-agent' stderr output:
debug2: fd 4 setting O_NONBLOCK

2022-04-27 18:53:33,490 DEBG 'ssh-agent' stderr output:
debug1: process_message: socket 1 (fd=4) type 17

2022-04-27 18:53:33,507 DEBG 'ssh-agent' stderr output:
debug1: process_add_identity: add ssh-rsa SHA256:g1TYNpJh/J5lofS6qRaapsSclKpUyl7TaT06k8BrY10 "django@354771a174ee" (life: 0) (confirm: 0) (provider: none)

2022-04-27 18:53:33,604 DEBG 'ssh-agent' stderr output:
debug2: fd 4 setting O_NONBLOCK
debug1: process_message: socket 1 (fd=4) type 11

2022-04-27 18:53:33,770 DEBG 'ssh-agent' stderr output:
debug2: fd 4 setting O_NONBLOCK

2022-04-27 18:53:33,857 DEBG 'ssh-agent' stderr output:
debug1: process_message: socket 1 (fd=4) type 17

2022-04-27 18:53:33,866 DEBG 'ssh-agent' stderr output:
debug1: process_add_identity: add ssh-rsa SHA256:g1TYNpJh/J5lofS6qRaapsSclKpUyl7TaT06k8BrY10 "django@354771a174ee" (life: 0) (confirm: 0) (provider: none)

2022-04-27 18:53:34,066 DEBG 'ssh-agent' stderr output:
debug2: fd 4 setting O_NONBLOCK

2022-04-27 18:53:34,074 DEBG 'ssh-agent' stderr output:
debug1: process_message: socket 1 (fd=4) type 11

2022-04-27 18:56:20,841 DEBG 'rqworker_low' stderr output:
WARNING - 2022-04-27 18:56:20,435 - environment - Failed to import module 'tf_detection_api_format.converter.py': The TensorFlow library was compiled to use AVX instructions, but these aren't available on your machine.
qemu: uncaught target signal 6 (Aborted) - core dumped


2022-04-27 18:56:21,039 DEBG 'rqworker_default_1' stderr output:
WARNING - 2022-04-27 18:56:20,481 - environment - Failed to import module 'tf_detection_api_format.converter.py': The TensorFlow library was compiled to use AVX instructions, but these aren't available on your machine.
qemu: uncaught target signal 6 (Aborted) - core dumped


2022-04-27 18:56:21,115 DEBG 'runserver' stderr output:
WARNING - 2022-04-27 18:56:20,972 - environment - Failed to import module 'tf_detection_api_format.converter.py': The TensorFlow library was compiled to use AVX instructions, but these aren't available on your machine.
qemu: uncaught target signal 6 (Aborted) - core dumped


2022-04-27 18:56:22,363 DEBG 'rqworker_default_0' stderr output:
WARNING - 2022-04-27 18:56:22,291 - environment - Failed to import module 'tf_detection_api_format.converter.py': The TensorFlow library was compiled to use AVX instructions, but these aren't available on your machine.
qemu: uncaught target signal 6 (Aborted) - core dumped


2022-04-27 18:56:32,088 DEBG 'rqworker_default_0' stderr output:
DEBUG - 2022-04-27 18:56:31,834 - worker - Registering birth of worker 1aead589e39b4b2a88e7a4bb647caa4d

2022-04-27 18:56:32,662 DEBG 'rqworker_default_0' stderr output:
INFO - 2022-04-27 18:56:32,651 - worker - Worker rq:worker:1aead589e39b4b2a88e7a4bb647caa4d: started, version 1.5.1

2022-04-27 18:56:32,672 DEBG 'rqworker_default_0' stderr output:
INFO - 2022-04-27 18:56:32,669 - worker - *** Listening on default...

2022-04-27 18:56:32,774 DEBG 'rqworker_default_0' stderr output:
DEBUG - 2022-04-27 18:56:32,713 - worker - Sent heartbeat to prevent worker timeout. Next one should arrive within 480 seconds.

2022-04-27 18:56:32,791 DEBG 'rqworker_default_0' stderr output:
INFO - 2022-04-27 18:56:32,773 - worker - Cleaning registries for queue: default

2022-04-27 18:56:32,823 DEBG 'rqworker_default_0' stderr output:
DEBUG - 2022-04-27 18:56:32,807 - worker - *** Listening on default...

2022-04-27 18:56:32,834 DEBG 'rqworker_default_0' stderr output:
DEBUG - 2022-04-27 18:56:32,821 - worker - Sent heartbeat to prevent worker timeout. Next one should arrive within 480 seconds.

2022-04-27 18:56:33,130 DEBG 'rqworker_default_1' stderr output:
DEBUG - 2022-04-27 18:56:33,109 - worker - Registering birth of worker 24d8dafd314049a9925800f4d91536aa

2022-04-27 18:56:33,224 DEBG 'rqworker_low' stderr output:
DEBUG - 2022-04-27 18:56:33,213 - worker - Registering birth of worker 2db9b0febe40466eb38a611f49346255

2022-04-27 18:56:33,351 DEBG 'rqworker_default_1' stderr output:
INFO - 2022-04-27 18:56:33,344 - worker - Worker rq:worker:24d8dafd314049a9925800f4d91536aa: started, version 1.5.1

2022-04-27 18:56:33,356 DEBG 'rqworker_default_1' stderr output:
INFO - 2022-04-27 18:56:33,355 - worker - *** Listening on default...

2022-04-27 18:56:33,396 DEBG 'rqworker_default_1' stderr output:
DEBUG - 2022-04-27 18:56:33,375 - worker - Sent heartbeat to prevent worker timeout. Next one should arrive within 480 seconds.

2022-04-27 18:56:33,456 DEBG 'rqworker_default_1' stderr output:
DEBUG - 2022-04-27 18:56:33,448 - worker - *** Listening on default...

2022-04-27 18:56:33,485 DEBG 'rqworker_default_1' stderr output:
DEBUG - 2022-04-27 18:56:33,476 - worker - Sent heartbeat to prevent worker timeout. Next one should arrive within 480 seconds.

2022-04-27 18:56:33,512 DEBG 'rqworker_low' stderr output:
INFO - 2022-04-27 18:56:33,511 - worker - Worker rq:worker:2db9b0febe40466eb38a611f49346255: started, version 1.5.1

2022-04-27 18:56:33,526 DEBG 'rqworker_low' stderr output:
INFO - 2022-04-27 18:56:33,522 - worker - *** Listening on low...

2022-04-27 18:56:33,533 DEBG 'rqworker_low' stderr output:
DEBUG - 2022-04-27 18:56:33,527 - worker - Sent heartbeat to prevent worker timeout. Next one should arrive within 480 seconds.

2022-04-27 18:56:33,537 DEBG 'rqworker_low' stderr output:
INFO - 2022-04-27 18:56:33,535 - worker - Cleaning registries for queue: low

2022-04-27 18:56:33,544 DEBG 'rqworker_low' stderr output:
DEBUG - 2022-04-27 18:56:33,543 - worker - *** Listening on low...

2022-04-27 18:56:33,546 DEBG 'rqworker_low' stderr output:
DEBUG - 2022-04-27 18:56:33,546 - worker - Sent heartbeat to prevent worker timeout. Next one should arrive within 480 seconds.

**Comment by merchantbreeze** [comment]
I also face this problem. Have you solved it?

**Comment by NormanTUD** [comment]
I also face this problem.

**Comment by bsekachev** [comment]
Could you please provide some screenshots and login response?

As I can see, there are might be some issues related with your CPU, it does not support AVX instructions:
```

**Comment by bsekachev** [comment]
I will close the issue. Do not hesitate to reopen if you can provide the requested information.

**Comment by NormanTUD** [comment]
For me, it didn't ever work and I must sadly say that I abandoned CVAT because of that. It looked very nice and I'm sure works great *when* it works, but installing it never did for me.
For anyone like me who's just interested in a free and quick solution I created a very simple PHP page that allows many people to annotate datasets via mouse with groups of people and then get the results. It's alpha still!, but for me it works better than something great that I cannot access for some technical reason.

https://github.com/NormanTUD/SimpleImageAnnotationTool

Maybe this helps someone someday.
